### ISSUE TITLE
Are there known solutions to the build problem referenced in issue #9

### ISSUE THREAD

**Issue by Calhistorian** [david-asher/electron-firebase#12]
I am having the same issue as the one mentioned in the referenced issue. Is there a known configuration requirement needed for the framework you have built here?

https://github.com/david-asher/electron-firebase/issues/9

For myself, I imported into Electron-Forge (which I have experience with and have built an early version of what I am building) and currently the project runs fine when started from the CL with electron-forge. Build command runs through fine - no errors. Zip is created. But the containing application immediately closes upon launch. 

I get this error in the macOS console:
![CleanShot 2021-12-08 at 22 21 40@2x](https://user-images.githubusercontent.com/51142180/145346059-9d36d054-6de9-46ad-b53c-c22e2ae04232.png)

Running: 
macOS 12.0.1

**Comment by Calhistorian** [comment]
To get at the root of the problem I chose to just go back to the template application in the module and build from there using my current firebase config project. Unfortunately, the executables/installers on macOS Monterey 12.0.1, Monterey 12.1 (Beta), and on Windows 10 (21H2) all failed to execute after packaging and build. This was done with Electron-Forge (import) and Electron-Builder. In all cases, the applications ran fine in a dev environment (when run with `npm start`). My own application as well as the template app after build failed to launch a window. This was on 3 different machines and 3 different operating systems. It seems clear that @jonybur was able to build the test application. 

To ensure that there wasn't an issue with my environment, I was still able to build 3 different electron applications on all three machines without problems. So its clear that electron-firebase has some kind of specific packaging requirements.

Is there a particular build configuration that you have used in the past to get this to work? Or what elements of the template could be causing the failures.

**Comment by Calhistorian** [comment]
Problem Solved. However, I think it important to record that the problem is with a particular variable that becomes `undefined` after packaging. And given some research, this is not an uncommon problem.

There are two instances of the variable that need to be changed in order for the test application to launch after packaging. 

**First**, in `mainapp.js` under the `getConfigFile()` function, `process.env.INIT_CWD` variable is called.
**Second**, in webserver.js` the same variable is called when starting the express server. In particular when defining the `staticFolder` paths: `appserver.use(slashFolder, express.static(process.env.INIT_CWD + slashFolder, folderOptions))` 

When launched from the command line with node (without packaging), this successfully results in the appropriate path. However, after packaging both of these instances come `undefined`. 

I don't claim to be an expert here, but this is how I solved it. And it may have been the solution found here too - [with issue #9](https://github.com/david-asher/electron-firebase/issues/9)

I switched the variable to the `getAppPath()` method of Electron's `app` module which resulted in the correct path for both dev and prod paths. 

Currently, I have only tested on macOS 12.1 with Node v16.13.1
